package.json

```diff
@@ -6,13 +6,23 @@
   "repository": "https://github.com/suda/toolbar",
   "license": "MIT",
   "keywords": [
-    "toolbar"
+    "toolbar",
+    "tool-bar"
   ],
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">=0.188.0"
   },
   "dependencies": {
-    "atom-space-pen-views": "^2.0.3",
+    "grim": "^1.2.2",
+    "space-pen": "^5.1.1",
     "underscore-plus": "^1.6.6"
+  },
+  "providedServices": {
+    "tool-bar": {
```

Comment:
The service is called `tool-bar` same as https://github.com/atom/status-bar/blob/master/package.json#L17